CefSharp.BrowserSubprocess.Core/CefAppUnmanagedWrapper.h

```diff
@@ -44,6 +48,8 @@ namespace CefSharp
             _extensions = gcnew List<CefExtension^>();
             _schemes = schemes;
             _focusedNodeChangedEnabled = enableFocusedNodeChanged;
+            _javascriptRootObject = gcnew ConcurrentDictionary<int, JavascriptRootObject^>();
```

Comment:
Variables need to be renamed e.g. `_javascriptRootObjects` as it's not a dictionary.